content/cloud/linking-applications-to-resources-using-labels.md

```diff
@@ -9,9 +9,11 @@ url = "https://github.com/fermyon/developer/blob/main/content/cloud/linking-appl
 
 - [Features of Links and Labels](#features-of-links-and-labels)
 - [Working With Links and Labels](#working-with-links-and-labels)
+  - [Linking Resources Between Applications](#linking-resources-between-applications)
+  - [Unlinking](#unlinking)
 - [Next Steps](#next-steps)
 
-Many Spin applications require data storage between invocations. Fermyon Cloud offers two methods for this: key-value storage and SQLite databases. However, the relationship between applications and databases isn't strictly one-to-one. A single database might be used by multiple applications, while an application could utilize several databases. There might even be a need to maintain a database that isn't utilized by any applications, perhaps for analytics or compliance reasons. All of these scenarios can occur as applications are independently deployed, upgraded, and retired, without any particular application needing to be aware of how others might be using the same databases.
+Many Spin applications require data storage between invocations. Fermyon Cloud offers two methods for this: Key Value storage and SQLite databases. However, the relationship between applications and databases isn't strictly one-to-one. A single database might be used by multiple applications, while an application could utilize several databases. There might even be a need to maintain a database that isn't utilized by any applications, perhaps for analytics or compliance reasons. All of these scenarios can occur as applications are independently deployed, upgraded, and retired, without any particular application needing to be aware of how others might be using the same databases.
```

Comment:
Have we capitalised this elsewhere?  I haven't been doing so, but I see you've updated this in a few places - is this "official" style now and should we be porting it across the docs and the UI?

Reply:
Not 100% sure TBH but I did a search for `Key Value` vs `key-value` and went with the majority wins. Happy to use Key Value moving forward, yeah.

Reply:
It was more the capitalisation - `key value` vs `Key Value`.  Agree we are inconsistent about the hyphenation